examples/sugarscape_g1mt/sugarscape_g1mt/model.py

```diff
@@ -0,0 +1,168 @@
+import numpy as np
+import math
+
+
+import mesa
+from .trader_agents import Trader
+from .resource_agents import Sugar, Spice
+
+# Helper Functions
+def flatten(list_of_lists):
+  '''
+  helper function for model datacollector for trade price
+  collapses agent price list into one list
+  '''
+  return [item for sublist in list_of_lists for item in sublist]
+
+def geometric_mean(list_of_prices):
+  '''
+  find the geometric mean of a list of prices
+  '''
+  return np.exp(np.log(list_of_prices).mean())
+
+def get_trade(agent):
```

Comment:
Just a note, not a change request: it'd be clearer with `get_trade_partners`, but if this is renamed, it would be harder for people who come from the CE video to associate `get_trade` with `get_trade_partners`.

Reply:
Fair point